client/src/Components/About/About.js

```diff
@@ -22,45 +23,48 @@ const About = () => {
   const user = useUser();
 
   const formatSection = (section, invertOrder) => {
-    const { title, paragraph, button } = section;
+    const { title, imageName, paragraph, button } = section;
 
     const getGridItem = (key, content) => {
       return (
-        <Grid item key={key} className={style.center} sm={6} xs={12}>
+        <Grid className={style.center} item key={key} sm={6} xs={12}>
           {content}
         </Grid>
       );
     };
 
     const titleGridItem = getGridItem(
-      title,
-      <Typography variant="h3" color="primary">
-        {title}
-      </Typography>
+      imageName,
+      <SvgIcon name={imageName} style={{ width: '65%', height: '65%' }} />
     );
 
-    const contentGridItem = getGridItem(button.href, [
-      <Typography key={button.content} gutterBottom variant="body1">
-        {paragraph}
-      </Typography>,
-      <Button
-        key={button.href}
-        variant="contained"
-        color="primary"
-        {...Object.assign(
-          {},
-          button.href.startsWith('/')
-            ? { onClick: () => history.push(button.href) }
-            : {
-                href: button.href,
-                target: '_blank',
-                rel: 'noopener noreferrer',
-              }
-        )}
-      >
-        {button.content}
-      </Button>,
-    ]);
+    const contentGridItem = getGridItem(
+      button.href,
```

Comment:
```suggestion
      title,
```

I think that now that we are using the titles here, it's better to use those as the keys.